src/views/GitBasics.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <basics-layout />
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+import BasicsLayout from "@/components/Basics/BasicsLayout.vue";
```

Comment:
@JuanJosePG  basics* Please review the inport or the name from the filters, cause in my pc there are problems with the folders name